content/vi/_index.html

```diff
@@ -41,12 +41,12 @@ <h2>Những thách thức của việc di chuyển hơn 150 Microservices lên K
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">Xem Video</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/" button id="desktopKCButton">Tham gia KubeCon Amsterdam 30/03-02/04/2020</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america" button id="desktopKCButton">Tham gia KubeCon Bắc Mỹ 24-28/10/2022</a>
```

Comment:
Please update the new event time: 06-09/11/2023

Reply:
ok, have updated both.